.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+### Describe the change
+
+Explanation of what this PR does
+
+### Where should the reviewer start?
+
+Recommendations for how to test this PR
+
+### PR checklist
+
+- [ ] Include tests (unit or E2E)
+
+### Browser checklist (only frontend):
+
+- [ ] Chrome
+- [ ] Firefox
+
+### Issue reference
+
+If this PR is a fix for a github issue, please link it here
```

Comment:
We can add the keywords Fixes or Closes

Reply:
I have added example with Closes keyword